llvm/include/llvm/CodeGen/AsmPrinter.h

```diff
@@ -385,10 +386,11 @@ class LLVM_ABI AsmPrinter : public MachineFunctionPass {
   /// are available. Returns empty string otherwise.
   StringRef getConstantSectionSuffix(const Constant *C) const;
 
-  /// Generate and emit labels for callees of the indirect callsites which will
-  /// be used to populate the .callgraph section.
-  void emitIndirectCalleeLabels(
-      FunctionInfo &FuncInfo,
+  /// Iff MI is an indirect call, generate and emit a label after the callsites
```

Comment:
is `Iff` a typo, or shorthand for "if and only if"? I can't tell from the context.

Reply:
Not a typo. This was intended.